dockerplugin.py

```diff
@@ -52,9 +53,35 @@ def _d(d):
     return ','.join(['='.join(p) for p in d.items()])
 
 
+def str_to_bool(value):
+    """Python 2.x does not have a casting mechanism for booleans.  The built in
+    bool() will return true for any string with a length greater than 0.  It
+    does not cast a string with the text "true" or "false" to the
+    corresponding bool value.  This method is a casting function.  It is
+    insensetive to case and leading/trailing spaces.  An Exception is raised
+    if a cast can not be made.
+    """
+    value = str(value).strip().lower()
+    if value == 'true':
+        return True
+    elif value == 'false':
+        return False
+    else:
+        raise Exception('Unable to cast value (%s) to boolean' % value)
+
+
 def emit(container, dimensions, point_type, value, t=None,
          type_instance=None):
     """Emit a collected datapoint."""
+    log.info(('Value parameters to be emitted: \n container : {c} '
+              '\n dimensions : {d} \n point_type : {pt} \nvalue : {v} '
```

Comment:
* You don't need the spaces before each `\n`.
* You are missing a space before `value :`.